openlibrary/templates/type/edition/modal_links.html

```diff
@@ -1,6 +1,6 @@
 $def with (work, edition, page_url)
 
-$def icon_link(link_class, img_src, link_text, login_redirect=True, ga_data=None):
+$def icon_link(link_class, img_src, link_text, login_redirect=True, ga_data=None, alt=""):
```

Comment:
```suggestion
$def icon_link(link_class, img_src, link_text, login_redirect=True, ga_data=None):
```
It isn't necessary to add `alt` here, as it will be the same value for each modal link.

Reply:
removing it from here is causing some issues on the page.

Reply:
Oh!  I see why this is happening.  The `alt` keyword argument needs to be removed from the `icon_link` call on line 28.  I overlooked that one during my review...

Let me test what you have now, and I can make a final commit to this branch that will remove the argument.